### PR TITLE
Update Towny to 1.14

### PR DESCRIPTION
**Pull request by creatorfromhell**
Some changes here for 1.14 compatibility. The large one is for the revert_on_unclaim feature as I had to update it to no longer use IDs... It now uses the recommended material keys. I haven't been able to test this on old data as I have no old data to test on. Here is the built jar for this PR: https://github.com/creatorfromhell/Towny/releases/tag/0.2.0.0

### CODE REVIEW

**Comment by LlmDl** [comment]
Nice stuff, I'll be looking at this over the long weekend (this weekend in Canada.) Old data revert-on-unclaim stuff should be as good as it gets, as we already have a system to read old plot snapshots into materials.

**Comment by creatorfromhell** [comment]
The latest commit, dubbed "I can't even." fixes an issue where boats bypass item_use protections due to the fact that the wrong material name was used in the default configuration file of towny.

**Comment by LlmDl** [comment]
> The latest commit, dubbed "I can't even." fixes an issue where boats bypass item_use protections due to the fact that the wrong material name was used in the default configuration file of towny.

Don't you mean "Even I Cant"

File: src/com/palmergames/bukkit/util/BukkitTools.java [hunk 1]
```diff
@@ -1,321 +1,323 @@
...
-	 * 
-	 * @param value
-	 * @return int of the relevant townblock x/z.
-	 */
-	public static int calcChunk(int value) {
-
-		return (value * TownySettings.getTownBlockSize()) / 16;
-	}
-}
+package com.palmergames.bukkit.util;
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/util/BukkitTools.java]: - Fix plot perm line not being set properly on plots owned by someone who leaves town.

Comment (LlmDl) [inline src/com/palmergames/bukkit/util/BukkitTools.java]: Did anything actually change in this file aside from the imports' order and the removal of a couple. 

Having all of these files get entirely nuked and then entirely re-added really makes reviewing a big PR like this a pain in the butt.

Comment (creatorfromhell) [reply src/com/palmergames/bukkit/util/BukkitTools.java]: There was the addition of a method: getTypeKey

Comment (LlmDl) [reply src/com/palmergames/bukkit/util/BukkitTools.java]: Any idea why github cannot tell why your commits aren't new classes?

Comment (creatorfromhell) [reply src/com/palmergames/bukkit/util/BukkitTools.java]: If I did, I would probably work at Github.

**Pull request closed without merging**